### DISCUSSION TITLE
[Feature Request]: Stop/start services individually

### DISCUSSION THREAD

**Discussion by ozanhazer** [beyondcode/herd-community#364] [Ideas]
### Feature Description

I have many projects and some of them need docker, so with valet, I was stopping nginx only and running docker since dnsmasq was still running, I could still use *.test addresses. Now since everything starts and stops together that's not possible.

The ability to start/stop services individually would provide flexibility for edge cases.

**Comment by flap152** [comment]
I'll add this use case instead of creating another request:
Using dusk or other means of "serving" an application (localhost:portXXX), I want to ensure app.test is not available, stopping nginx but not the other services.
